editor/src/clj/util/eduction.clj

```diff
@@ -13,7 +13,14 @@
 ;; specific language governing permissions and limitations under the License.
 
 (ns util.eduction
-  (:refer-clojure :exclude [dedupe distinct drop drop-while filter interpose keep keep-indexed map map-indexed mapcat partition-all partition-by random-sample remove replace take take-nth take-while]))
+  (:refer-clojure :exclude [concat dedupe distinct drop drop-while filter interpose keep keep-indexed map map-indexed mapcat partition-all partition-by random-sample remove replace take take-nth take-while]))
+
+(defn concat
+  ([] [])
+  ([x] x)
+  ([x y] (eduction cat [x y]))
+  ([x y & zs]
+   (eduction cat (into [x y] zs))))
```

Comment:
I think as a general rule, we should strive to add tests for these low-level utility functions that we add to the `util` namespace. However, I also implemented `e/concat` on one of my branches and have already written some tests for it, so it will be covered eventually. 😅

Reply:
I added a test btw: https://github.com/defold/defold/pull/9613/files#diff-b969c5c1820d51ab1793534439a214ef7c849ec7c9bbb9a29f15cf0e01154e5c
I'll test the build on windows and linux since there are platform-specific path handling, and then I'll merge it if everything's ok.